com.unity.render-pipelines.universal/Runtime/Passes/ColorGradingLutPass.cs

```diff
@@ -62,113 +63,116 @@ public void Setup(in RenderTargetHandle internalLut)
         public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
         {
             var cmd = CommandBufferPool.Get(k_ProfilerTag);
```

Comment:
should this be nameless?

Reply:
It should and I'm pretty sure I changed that, but I guess I undid that.

Reply:
Fixed